contrib/hawq-hadoop/hawq-mapreduce-common/src/main/java/com/pivotal/hawq/mapreduce/metadata/MetadataYAMLAccessor.java

```diff
@@ -198,6 +201,9 @@ private HAWQParquetTableMetadata extractParquetMetadata(HAWQDatabaseMetadata dbM
 			Map<?, ?> file = (Map<?, ?>) files.get(i);
 			final String filePath = file.get("path").toString();
 			final long fileLen = Long.parseLong(file.get("size").toString());
+			if (fileLen == 0) {
+			  throw new IllegalStateException("Could not new HAWQParquetFileStatus with a empty file.");
+			}
```

Comment:
Could we change the error message to a more meaningful description.  For example, "Empty file $filePath, can not be processed by HAWQParquetInputFormat"